safe_transaction_service/history/services/index_service.py

```diff
@@ -320,12 +321,17 @@ def _reprocess(self, addresses: List[str]):
         queryset.delete()
 
         logger.info("Remove Safe statuses")
-
         queryset = SafeStatus.objects.all()
         if addresses:
             queryset = queryset.filter(address__in=addresses)
         queryset.delete()
 
+        logger.info("Remove Safe Last statuses")
+        queryset = SafeLastStatus.objects.all()
+        if addresses:
+            queryset = queryset.filter(address__in=addresses)
+        queryset.delete()
```

Comment:
This is intended? here you can remove all lastStatus rows.

Reply:
Same as for the other elements, right?

Reply:
yes :)